package.json

```diff
@@ -1,17 +1,10 @@
 {
-  "name": "CrackingJS",
-  "version": "0.0.2",
-  "description": "Data Structures and Algorithms in Javascript",
-  "scripts": {
-    "lint": "eslint lib/**/*.js test/**/*.js",
-    "test": "npm run lint && mocha --recursive"
-  },
-  "dependencies": {
-    "chai": "*",
-    "eslint": "^2.2.0",
-    "mocha": "*",
-    "require-directory": "*",
-    "sinon": "*"
-  },
-  "devDependencies": {}
+  "name": "CtCI",
+  "version": "0.0.1",
+  "description": "Cracking the Coding Interview - Answers in Javascript",
+  "scripts": {},
+  "dependencies": {},
```

Comment:
You removed linting and tests entirely. I can get on board with doing the `console.log` method of testing instead of chai, but keep `eslint`. It helps prevent a whole class of potential issues.